canary/webrtc-c/src/Include.h

```diff
@@ -95,6 +96,7 @@
 #define KVS_METRICS_INVOCATION_PERIOD        (5 * HUNDREDS_OF_NANOS_IN_A_SECOND)
 #define CANARY_METADATA_SIZE                 (SIZEOF(UINT64) + SIZEOF(UINT32) + SIZEOF(UINT32))
 
+#define STORAGE_CANARY_METADATA_SIZE         (SIZEOF(UINT64) + SIZEOF(UINT32) + SIZEOF(UINT32) + SIZEOF(UINT32))
```

Comment:
What is the additional SIZEOF(UINT32)) for? Also, I think we can remove custom frames related code altogether for this sample since we know that does not work.

Reply:
Yes sorry, meant to remove that